#3827 whitespace tokenizer (#4020)
RasaHQ/rasa: CHANGELOG.rst

Added
-----

> - WhiteSpaceTokenizer option added with an option for case_sensitive to be true or false, true being default.

Reviewer: let's rephrase:
```suggestion
- ``case_sensitive`` option added to ``WhiteSpaceTokenizer`` with ``true`` is default.
```

Author: done
